ui/lib/core/addon/components/form-field.hbs

```diff
@@ -83,16 +83,15 @@
         @models={{@attr.options.models}}
         @onChange={{this.setAndBroadcast}}
         @inputValue={{get @model this.valuePath}}
-        @helpText={{@attr.options.helpText}}
-        @subText={{@attr.options.subText}}
         @wildcardLabel={{@attr.options.wildcardLabel}}
         @label={{this.labelString}}
-        @subLabel={{@attr.options.subLabel}}
+        @labelClass={{if @attr.options.isNotSectionHeader "is-label" "title is-4"}}
```

Comment:
I wonder if we should instead use `isSectionHeader` and default to false, especially since it seems like using it as section header is an old and unpopular approach at this point

Reply:
I wondered the same! I was just nervous to change the default, but I can dig around and find all the instances it's intentionally used _as_ a header - shouldn't be too hard, I just have to track down all model attrs with `editType: 'search-select'`